spec/triggers/post_created_edited_spec.rb

```diff
@@ -97,6 +97,18 @@
         expect(list[0]["kind"]).to eq("post_created_edited")
       end
 
+      context "when the topic is not a PM" do
+        it "doesn’t fire the trigger" do
+          list =
+            capture_contexts do
+              user.groups << target_group
+              PostCreator.create(user, basic_topic_params)
```

Comment:
No need to make a change, but I do find it slightly difficult to read when I have to refer back to the top for what `basic_topic_params` means. In some cases, I would favour duplication or perhaps a more explicit variable name like `public_topic`